src/backend/distributed/commands/multi_copy.c

```diff
@@ -1217,6 +1228,209 @@ ReportCopyError(MultiConnection *connection, PGresult *result)
 }
 
 
+static void
+ConversionPathForTypes(Oid inputType, Oid destType, CopyCoercionData *result)
+{
+	Oid coercionFuncId = InvalidOid;
+	CoercionPathType coercionType;
+
+	if (destType == inputType)
```

Comment:
I guess it doesn't really matter, since we can use whichever value we'd like here, but `COERCION_PATH_NONE` means [conversion isn't possible](https://github.com/postgres/postgres/blob/REL_10_0/src/include/parser/parse_coerce.h#L26). And technically, a type _is_ binary castable to itself, so `RELABLE` isn't the wrong thing to send.